plugins/modules/apt_rpm.py

```diff
@@ -102,6 +123,26 @@ def update_package_db(module):
     if rc != 0:
         module.fail_json(msg="could not update package db: %s" % err)
 
+    return out
+
+
+def clean(module):
+    rc, out, err = module.run_command("%s clean" % APT_PATH)
+
+    if rc != 0:
+        module.fail_json(msg="apt-get clean failed: %s" % err)
```

Comment:
Instead of manually checking the return code, simply add `check_rc=True`. Then AnsibleModule.run_command provides much more information to the user in case the command fails.